packages/legend-query/src/stores/QueryBuilderFilterState.ts

```diff
@@ -139,6 +140,12 @@ export class FilterConditionState {
     );
   }
 
+  get propertyExpression(): QueryBuilderPropertyExpressionState {
```

Comment:
why do we do this? people can use `propertyExpression` from `this.propertyExpressionState.propertyExpression`. Doing something like this is confusing. If you meant to override the `propertyExpressionState` with a version that is equiped with `Milestoning` stuffs, you can probably do it right in the constructor.